dali/sasha/saserver.cpp

```diff
@@ -322,6 +323,9 @@ int main(int argc, const char* argv[])
         lf->setMaxDetail(TopDetail);
         lf->beginLogging();
     }
+#else
+    setupContainerizedLogMsgHandlerFromConfig();
```

Comment:
this is fine, just note to myself really, that if this is were called now, it would throw an exception in queryComponentConfig(), because Sasha doesn't use the new init system yet (but it should).
But since Sasha isn't yet containerized either - it will never be called.

It's fine though, since this is what we want when it is, so I'd leave it.